track/__init__.py

```diff
@@ -16,6 +16,17 @@ def create_app(environment='development'):
     else:
         app.config.from_object('track.config.ProductionConfig')
     app.config.from_pyfile('application.cfg', silent=True)
+
+    import logging
```

Comment:
Might want to pull the logging stuff out into it's own separate PR.

Reply:
I was starting to think about that as well, seemed like it was getting a bit scattered.